src/assets/index.css

```diff
@@ -377,6 +212,19 @@
 #chart-leftarea {
   margin-left: 3%;
 }
+#calories,#step,#water,#sleep {
+  display: none;
+  border-radius: 4px;
+  cursor: pointer;
+  overflow: visible !important;
```

Comment:
Try to avoid important here

Reply:
Avoided unnecessary usage of !important tags